include/deal.II/base/mpi.h

```diff
@@ -1027,6 +1030,31 @@ namespace Utilities
       static std::set<MPI_Request *> requests;
     };
 
+    namespace internal
+    {
+      template <typename T>
+      constexpr bool
+      is_mpi_type()
```

Comment:
Let's add documentation to this function:
```suggestion
      /**
       * Return `true` if the type `T` is a type that MPI understands natively and for which send and
       * receive operations are supported by the MPI standard.
       */
      template <typename T>
      constexpr bool
      is_mpi_type()
```